docs/data/date-pickers/date-time-picker/ResponsiveDateTimePickers.tsx

```diff
@@ -10,7 +10,7 @@ import { Unstable_DesktopNextDateTimePicker as DesktopNextDateTimePicker } from
 export default function ResponsiveDateTimePickers() {
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
-      <Stack spacing={3}>
+      <DemoContainer>
```

Comment:
Nitpick again, but this demo could as well use a bit more width.

Reply:
See https://github.com/mui/mui-x/pull/7149#discussion_r1045490074